### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by agarzon48** [Antoniomorales17/To-DoList#1]
Buen trabajo, Antonio! Podrías haber mejorado la presentación empaquetándola dentro de un div centrado o con un poco más de estilos, pero perfecta la app

**Comment by Antoniomorales17** [comment]
Cierto, los estilos siempre quedan en segundo plano jajajaja Gracias Adrián, a seguir!
